CONTRIBUTING.md

```diff
@@ -29,9 +29,9 @@ Go in the repo and run maven:
     cd dhis-web
     mvn install -U
 
-Each project in the /dhis-2/dhis-web directory is an individual web module. The dhis-web-portal project is an assembly of all the individual web modules.
+Each project in the /dhis-2/dhis-web directory is an individual web module. The dhis-web-server project is an assembly of all the individual web modules.
```

Comment:
Just remove this part "Each project in the /dhis-2/dhis-web directory is an individual web module."